packages/salesforcedx-vscode-core/src/commands/util/conflictDetectionMessages.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
```

Comment:
So for these two exports they were originally in `postconditionCheckers.ts` with a ton of other stuff.  This lead to always importing this metadata caching checker that had a ton of dependancies and eventually had issues.  Simplified by moving these out to their own files and re-exporting from there for index.ts.